server/halomod_app/utils.py

```diff
@@ -38,67 +40,70 @@ def hmf_driver(cls=TracerHaloModel, previous: Union[None, TracerHaloModel] = Non
         return this
 
 
-def create_canvas(objects, q: str, d: dict, plot_format: str = "png"):
+def create_canvas(objects, x_key: str, y_key: str, plot_format: str = "png"):
+    KEYMAP = load_json('../globals/plotData.json')
```

Comment:
It might be good to change the name from KEYMAP to something else to help differentiate. Like `PLOT_DATA` or something along those lines.

Reply:
this has been removed